src/modules/Common/pages/user/[user].staticPaths.ts

```diff
@@ -0,0 +1,16 @@
+/* import dbConnect from 'utils/db';
+import * as HashtagManager from '../../managers/HashtagManager'; */
+import { GetStaticPaths } from 'next';
+
+export const getStaticPaths: GetStaticPaths<{
+  hashtag: string;
+}> = async () => {
+  /* await dbConnect();
+  const hashtags = await HashtagManager.list(); */
```

Comment:
should have `const users = await UserManager.list();` to generate users pages on build

See https://nextjs.org/docs/basic-features/data-fetching